scss/_tables.scss

```diff
@@ -25,8 +26,7 @@
   // stylelint-disable-next-line selector-max-universal
   > :not(caption) > * > * {
     padding: $table-cell-padding-y $table-cell-padding-x;
-    background-color: var(--bs-table-bg);
-    background-image: linear-gradient(var(--bs-table-accent-bg), var(--bs-table-accent-bg));
+    background-color: $table-bg;
```

Comment:
In this file we use only SCSS variables in style properties values, every - without expressions, just variables.